RimWorld/RimWorld/Tool.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <SFML/Graphics.hpp>
+#include "ResourceManager.h"
+#include "NpcManager.h"
+#include "Map.h"
```

Comment:
I suppose all of these includes can be moved to cpp and corresponding classes just be forward declared for arguments  in declarations